ERROR message for each service.

Does cloudflare-companion do any tracking or checks before it looks to create DNS records?

I keep getting this in my logs for each service whenever I restart my docker-compose:
`today at 18:48:36 [ERROR] ** dozzle.host.com - 81053 An A, AAAA, or CNAME record with that host already exists.`

My config:
```
  cf-companion:
    container_name: cf-companion
    image: tiredofit/traefik-cloudflare-companion:latest
    restart: unless-stopped
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
    environment:
      TRAEFIK_VERSION: 2
      CF_EMAIL: $CLOUDFLARE_EMAIL
      CF_TOKEN: $CLOUDFLARE_API_KEY
      TARGET_DOMAIN: $DOMAINNAME
      DOMAIN1: $DOMAINNAME
      DOMAIN1_ZONE_ID: $CLOUDFLARE_ZONE_ID
      DOMAIN1_PROXIED: "TRUE"
      PUID: $PUID
      PGID: $PGID
      TZ: $TZ
      UMASK: $UMASK
```

It's not a major issue, just not sure if it should be an `ERROR`

Sorry if I've missed something!

--- a/install/usr/sbin/cloudflare-companion
+++ b/install/usr/sbin/cloudflare-companion
@@ -204,11 +204,15 @@ def point_domain(name, domain_infos):
                     pass
             else:
                 try:
-                    if DRY_RUN:
-                        logger.info("DRY-RUN: POST to Cloudflare %s:, %s", domain_info['zone_id'], data)
-                    else:
-                        _ = cf.zones.dns_records.post(domain_info['zone_id'], data=data)
-                    logger.info("Created new record: %s to point to %s", name, domain_info['target_domain'])
+                    for record in records:
+                        if record['content'] != domain_info['target_domain']:
+                            if DRY_RUN:
+                                logger.info("DRY-RUN: POST to Cloudflare %s:, %s", domain_info['zone_id'], data)
+                            else:
+                                _ = cf.zones.dns_records.post(domain_info['zone_id'], data=data)
+                            logger.info("Created new record: %s to point to %s", name, domain_info['target_domain'])
+                        else:
+                            logger.info("Record already exists: %s points %s", name, domain_info['target_domain'])
                 except CloudFlare.exceptions.CloudFlareAPIError as ex:
                     logger.error('** %s - %d %s' % (name, ex, ex))
                     ok = False

